File: Chitunion/Web(code refactoring)/XYAuto.ChiTu2018/XYAuto.ChiTu2018.Entities/Chitunion2017/User/UserRole.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace XYAuto.ChiTu2018.Entities.Chitunion2017.User
{
    [Table("UserRole")]
    public partial class UserRole
    {
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int RecID { get; set; }

        [Key]
        [Column(Order = 0)]
        [DatabaseGenerated(DatabaseGeneratedOption.None)]
        public int UserID { get; set; }

        [Key]
        [Column(Order = 1)]
        [StringLength(50)]
        public string RoleID { get; set; }

        [Required]
        [StringLength(50)]
        public string SysID { get; set; }

        public int? Status { get; set; }

        public DateTime? CreateTime { get; set; }

        public int? CreateUserID { get; set; }
    }
}

File: Chitunion/Web(java)/src/main/java/com/xyauto/domain/UserRole.java
package com.xyauto.domain;

import java.util.Date;

import lombok.Data;

@Data
public class UserRole {

	private Integer userID;
	private String roleID;
	private Integer recID;
	private String sysID;
	private Integer status;
	private Date createTime;
	private Integer createUserID;
	private String newRoleID;
}